BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -13,9 +13,13 @@ public protocol StackElement: Element {
 extension StackElement {
 
     public var content: ElementContent {
-        return ElementContent(layout: layout) {
+        ElementContent(layout: layout) {
```

Comment:
Our Package.swift specifies `.v5` as the Swift language version, which doesn’t include implicit `return`s. But there doesn’t seem to be a `.v5_1`. We should probably either bump to `.version("5.1")` or not use 5.1-only features?

Reply:
Is "5.1" a real thing? AFAIK, Xcode only thinks in terms of major version, right?

Reply:
Not exactly sure TBH. But [there’s a `.v4_2`](https://github.com/apple/swift-package-manager/blob/master/Sources/PackageDescription/LanguageStandardSettings.swift#L94)

Reply:
We could set the `swift-tools-version` magic comment to 5.1, but from what I understand the 5.1 enum case isn't there because it's not really a thing you can enforce, and there are only going to be cases for major versions going forward. This also came up when I specified 5.1 in the podspecs, because apparently `swiftc` also won't accept anything above major version 5.